test/e2e/cluster.sh

```diff
@@ -353,3 +353,19 @@ if [ "${SCALE_CLUSTER}" = "true" ]; then
     -e ADD_NODE_POOL_INPUT=${ADD_NODE_POOL_INPUT} \
     ${DEV_IMAGE} make test-kubernetes || exit 1
 fi
+
+if [ "${GET_CLUSTER_LOGS}" = "true" ]; then
```

Comment:
O.K., so actually, to make this easier to test, let's move this condition into the condition in L115

And then add the actual get-logs statement after the RESOURCE_GROUP and REGION vars have been created, but before the random tags foo.

Basically, by doing that, we don't have to set upgrade/scale/addpool to true in order to test get-logs.